.gitignore

```diff
@@ -1,45 +1,5 @@
-*.py[cod]
```

Comment:
Time out.  A large portion of this stuff being removed appears to be relevant to me, and the remainder may just be because I have relatively little experience hacking in this repo.